src/test/cypress/e2e/exam/test-exam/TestExamCreationDeletion.cy.ts

```diff
@@ -0,0 +1,110 @@
+import { Interception } from 'cypress/types/net-stubbing';
+import { Course } from 'app/entities/course.model';
+import { CypressExamBuilder, convertCourseAfterMultiPart } from '../../../support/requests/CourseManagementRequests';
+import dayjs from 'dayjs/esm';
+import { artemis } from '../../../support/ArtemisTesting';
+import { dayjsToString, generateUUID, trimDate } from '../../../support/utils';
+
+// Requests
+const courseManagementRequests = artemis.requests.courseManagement;
+
+// Pageobjects
+const navigationBar = artemis.pageobjects.navigationBar;
+const courseManagement = artemis.pageobjects.course.management;
+const examManagement = artemis.pageobjects.exam.management;
+const creationPage = artemis.pageobjects.exam.creation;
+const examDetailsPage = artemis.pageobjects.exam.details;
+
+const examData = {
+    title: 'exam' + generateUUID(),
+    visibleDate: dayjs(),
+    startDate: dayjs().add(1, 'day'),
+    endDate: dayjs().add(2, 'day'),
+    workingTime: 5,
+    numberOfExercises: 4,
+    maxPoints: 40,
+    startText: 'Cypress exam start text',
+    endText: 'Cypress exam end text',
+    confirmationStartText: 'Cypress exam confirmation start text',
+    confirmationEndText: 'Cypress exam confirmation end text',
+};
+
+describe('Test Exam creation/deletion', () => {
+    let course: Course;
+    let examId: number;
+
+    before(() => {
+        cy.login(artemis.users.getAdmin());
+        courseManagementRequests.createCourse().then((response) => {
+            course = convertCourseAfterMultiPart(response);
+        });
+    });
```

Comment:
Since this is used in many other test suits as well, should we create a common function for that?

Reply:
Good point! Since I refactor the E2E tests anyway I would like to add this in a followup PR if this is alright with you